boot/zephyr/flash_map_extended.c

```diff
@@ -1,6 +1,7 @@
 /*
  * Copyright (c) 2018 Nordic Semiconductor ASA
  * Copyright (c) 2015 Runtime Inc
+ * Copyright (c) 2019 Arm Limited
```

Comment:
This is more in line with how I think we should be doing the copyrights. I would recommend participants update the date ranges when they make modifications, although that is more an issue for the individual contributors to decide. And in this particular instance, it is debatable whether this constitutes a copyrightable work, but having the copyright lines should be a problem.